app/components/moderation-list/template.hbs

```diff
@@ -0,0 +1,57 @@
+                        <button type="button" class="btn btn-filter" data-target="accepted" onclick={{action "buttonPressed" 'accepted'}}><a class="fa fa-check-circle-o"></a> {{model.acceptedPreprints.meta.pagination.total}} {{t 'moderation_list.accepted'}}</button>
+                        <button type="button" class="btn btn-filter" data-target="rejected" onclick={{action "buttonPressed" 'rejected'}}><a class="fa fa-times-circle-o"></a> {{model.rejectedPreprints.meta.pagination.total}} {{t 'moderation_list.rejected'}}</button>
+                        <div class="dropdown pull-right">
+                            <button class="btn dropdown-toggle" type="button" id="sortBy" data-toggle="dropdown">
+                                {{t 'moderation_list.sort'}}
+                                <span class="fa fa-caret-down"></span>
+                            </button>
+                            <ul class="dropdown-menu dropdown-menu-right" id="sortByOptionList">
+                                <span>
+                                    <button class="btn" onclick={{action "sortSelected" '-date_created'}}><a class='{{if (eq sort '-date_created') 'fa fa-check'}}'></a> {{t 'moderation_list.newest'}}</button>
+                                    <button class="btn" onclick={{action "sortSelected" 'date_created'}}><a class='{{if (eq sort 'date_created') 'fa fa-check'}}'></a> {{t 'moderation_list.oldest'}}</button>
+                                </span>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                {{#if loading}}
+                    <div class="text-center p-v-md" aria-label='Loading data'>
+                        {{fa-icon 'spinner' ariaHidden=false pulse=true size=3}}
+                    </div>
+                {{else}}
+                    <div class="panel-body">
+                        <div class="table-container">
+                            {{#if (eq buttonType 'accepted')}}
+                                {{accepted-record model=model screenWidth=screenWidth}}
+                            {{else if (eq buttonType 'pending')}}
+                                {{pending-record model=model screenWidth=screenWidth}}
+                            {{else}}
+                                {{rejected-record model=model screenWidth=screenWidth}}
+                            {{/if}}
+                        </div>
+                    </div>
+                    {{#if (and (gt model.pendingPreprints.meta.total 1) (eq buttonType 'pending'))}}
+                        <div class="pull-right text-right">
+                            {{pagination-pager count=model.pendingPreprints.meta.total current=pendingpage change=(action 'pageChanged')}}
```

Comment:
I think using a computed property here instead of switching in the template would be nicer